docs/source/algorithms/EstimateResolutionDiffraction-v1.rst

```diff
@@ -30,11 +30,12 @@ and thus
 
 where,
 
--  :math:`\Delta T` is the time resolution from moderator
--  :math:`\Delta\theta` is the coverage of the detector, and can be
+-  :math:`\Delta T` is the time resolution, generally from moderator dimensions. If supplied through ``DeltaTOF`` parameter, the center wavelength is used to calculate the center time-of-flight. Alternatively, one can specify ``DeltaTOFOverTOF``.
+-  :math:`\Delta\theta` is the uncertainty in size due to the source size and coverage of the detector. The former is supplied through the ``SourceDeltaTheta`` parameter. The latter can be
```

Comment:
:math:`\Delta\theta` is the uncertainty in **angle** due to the source size and coverage of the detector. The former is supplied through the ``SourceDeltaTheta`` parameter.